src/libraries/System.Private.CoreLib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -1055,7 +960,7 @@ public bool Remove(TKey key, [MaybeNullWhen(false)] out TValue value)
                 {
                     ref Entry entry = ref entries[i];
 
-                    if (entry.hashCode == hashCode && (_comparer?.Equals(entry.key, key) ?? EqualityComparer<TKey>.Default.Equals(entry.key, key)))
+                    if (entry.hashCode == hashCode && (comparer?.Equals(entry.key, key) ?? EqualityComparer<TKey>.Default.Equals(entry.key, key)))
```

Comment:
Add the IsValueType optimization?

Reply:
I do not see this comment addresses. Have you decided that the IsValueType optimization is not worth it in this particular place? Just curious.

Reply:
Did I miss one? Isn't this https://github.com/dotnet/runtime/pull/75663/files#diff-e5e062feb7b078d66f972ed4645eecbe12166633901eccceb847515a7e7d4a28R965?